src/services/swap_service.ts

```diff
@@ -266,6 +267,11 @@ export class SwapService implements ISwapService {
         } = params;
 
         let _rfqt: GetMarketOrdersRfqOpts | undefined;
+
+        if (rfqt?.nativeExclusivelyRFQ && !this._rfqClient.isRfqtEnabled()) {
+            throw new ServiceDisabledError('RFQt Service is not available');
+        }
```

Comment:
I believe `nativeExclusivelyRFQ` will only be true when the included sources has `RFQT` but not `Native`, `Mesh`, or `0x`. 

So, for example, it will fire if `includedSources=UniswapV3,RFQT` but not `includedSources=UniswapV3,0x`


Is the intent to fire when `rfqt` is enabled?